Drop after lang/rust >= 1.74 update

error[E0658]: use of unstable library feature 'io_error_other'
   --> common/assets/src/tar_source.rs:147:42
    |
147 |                     entry.path().map_err(io::Error::other)?.as_ref(),
    |                                          ^^^^^^^^^^^^^^^^
    |
    = note: see issue #91946 <https://github.com/rust-lang/rust/issues/91946> for more information
    = help: add `#![feature(io_error_other)]` to the crate attributes to enable

--- common/assets/src/lib.rs.orig	2023-10-18 12:03:13 UTC
+++ common/assets/src/lib.rs
@@ -1,4 +1,5 @@
 //#![warn(clippy::pedantic)]
+#![feature(io_error_other)]
 //! Load assets (images or voxel data) from files
 
 use dot_vox::DotVoxData;
